### PR TITLE
Added CreateBlock and CreateHead functions to new file

### PR DESCRIPTION
**Pull request by dipack95**
Exposing `CreateBlock`, `CreateHead` functions from #5887 as a new PR - per @krasi-georgiev 's request.

### CODE REVIEW

File: tsdb/tsdbblockutil.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	"fmt"
+	"github.com/go-kit/kit/log"
+	tsdb_labels "github.com/prometheus/prometheus/tsdb/labels"
+	"os"
+	"path/filepath"
+)
+
+// Default duration of a block in milliseconds - 2h.
+const (
+	DefaultBlockDuration = int64(2 * 60 * 60 * 1000)
```
Comment (krasi-georgiev) [inline tsdb/tsdbblockutil.go]: If we are introducing this const lets set it in the main db.go file and use it in `ExponentialBlockRanges`

Comment (dipack95) [reply tsdb/tsdbblockutil.go]: You mean you change this line: https://github.com/prometheus/prometheus/blob/de0a772b8e7d27dc744810a1a693d97be027049a/tsdb/db.go#L51

to be `BlockRanges:     DefaultBlockDuration`?

Comment (dipack95) [reply tsdb/tsdbblockutil.go]: Uh, sorry, I misunderstood. You want to use it like so: `ExponentialBlockRange(DefaultBlockDuration, 3, 5)`

**Comment by dipack95** [comment]
Odd, both the CircleCI test and Travis builds broke on unrelated parts of this PR.

**Comment by krasi-georgiev** [comment]
maybe try to rebase.

**Comment by dipack95** [comment]
Well I rebased this and #5887 to upstream master, and they built + tested fine on my local, but the CircleCI pipelines seem to have some trouble fetching the docker images.

**Comment by dipack95** [comment]
@krasi-georgiev All checks are green now!

**Pull request merged**